### ISSUE TITLE
Is it possible to not create output zip if there are no files in specified paths when using grunt-contrib-compress?

### ISSUE THREAD

**Issue by sharikovvladislav** [gruntjs/grunt-contrib-compress#185]
Hi,

I have this target:

```
module.exports = function (grunt, config) {
  var path = require('path'),
    zipOutputPath = /*getOutputPath*/,
    someOptions = /*getOptions*/;

  return {
    options: someOptions,
    myTarget: {
      options: {
        archive: zipOutputPath + path.join + 'zipName.zip'
      },
      files: [
        {
          src: ['test-reports/**']
        },
        {
          src: ['*.xml']
        }
      ]
    }
  };
};
```

Imagine there are no files in `test-reports/**` path and no `xml` files in cwd. After running `grunt compress:myTarget` `grunt-contrib-compress` will create **empty** zip archive with name `zipName` at `zipOutputPath`.

Is it somehow possible to not create zip archive if there were no files in specified places? Maybe there is some option or any workaround available?

**Comment by sharikovvladislav** [comment]
I created this PR to get this feature: https://github.com/gruntjs/grunt-contrib-compress/pull/186

**Comment by sharikovvladislav** [comment]
Implemented in `grunt-contrib-compress@1.4.3`
